test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -69,6 +69,22 @@ Describe 'Basic Job Tests' -Tags 'CI' {
             $ProgressMsg[0].StatusDescription | Should -BeExactly 2
         }
 
+        It 'Can use the user specified working directory parameter' {
+            $tempPath = [System.IO.Path]::GetTempPath()
```

Comment:
Can we use `$TestDrive` instead of `[System.IO.Path]::GetTempPath()`. Some platforms do not have a temp location defined.

Reply:
I did not know about this Pester feature. This pretty cool. Thanks. fixed in 62a7b0c71ac60747f84ffba8f523ff0004fd17c9